src/DroneSafetyApi/Startup.cs

```diff
@@ -53,9 +53,15 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMvc();
 
-            //services.Configure<DbOptions>(Configuration.GetSection("DocumentDB"));
-            //services.AddSingleton<IHazardRepository, HazardDatabase>();
-            services.AddSingleton<IHazardRepository, ExampleHazardRepository>();
+            services.Configure<DbOptions>(Configuration);
+            if (Configuration.GetValue<string>("EndpointUri") == null)
+            {
+                services.AddSingleton<IHazardRepository, ExampleHazardRepository>();
+            }
+            else
+            {
+                services.AddSingleton<IHazardRepository, HazardDatabase>();
+            }
```

Comment:
I checked this on my machine, the problem is that the cast to int in calculating the radius has been changed in the the actual class but not in the test class (in the pythogrean) method. I recommend to take Josh's approach and change it in the test because it gives a bigger radius and so it gives a better guarantee of including all the hazards